packages/styles/src/textarea.scss

```diff
@@ -24,6 +24,15 @@ $block: #{$fd-namespace}-textarea;
   display: block;
   cursor: text;
 
+  // &[readonly],
+  // &.is-readonly {
+  //   font-size: var(--sapFontSmallSize);
+  //   color: var(--sapField_TextColor);
+  //   border-radius: var(--sapField_BorderCornerRadius);
+  //   background-color: var(--sapField_ReadOnly_Background);
+
+  // }
```

Comment:
if we don't use maybe remove? Or if we decide to keep add comments on why it's commented out.

Reply:
this was a mistake, thanks